robotiq_description/urdf/2f_85.ros2_control.xacro

```diff
@@ -44,7 +44,7 @@
 
             <!-- Joint interfaces -->
             <!-- With Gazebo or Hardware, they handle mimic joints, so we only need this command interface activated -->
-            <joint name="${prefix}robotiq_85_left_knuckle_joint">
+            <joint name="${prefix}left_knuckle_joint">
```

Comment:
These changes would break existing setups, right? Should we update the default `prefix`  value to `robotiq_85_`?